databases/migration/README.md

```diff
@@ -0,0 +1,29 @@
+# UnifiedPush Server - Database migration
+
+For the database migration you may need to run a few scripts inside of the database
+
+## 1.0.0 users
+
+The UnifiedPush Server is supporting two different databases, MySQL and PostgreSQL, below you find information on how to perform the migration on each of the supported databases
+
+### MySQL
+
+Inside of the database execute these two scripts:
+
+* `keycloak-migration-mysql.sql`
+* `ups-migration-mysql.sql`
+
+### PostgreSQL
+
+Inside of the database execute these two scripts:
```

Comment:
Should mention that the order is important (first KC and then UPS or is the order not relevant ?